test/unit/spec/v2/ion/uiSchemaTransformer_spec.js

```diff
@@ -763,6 +789,14 @@ describe('v2/ion/uiSchemaTransformer', function() {
                     authenticatorKey: 'symantec_vip',
                     relatesTo: XHRAuthenticatorEnrollSelectAuthenticators.authenticators.value[11]
                   },
+                  {
+                    label: 'YubiKey Authenticator',
```

Comment:
The label here shows up in select authenticator list UI and this doesnt seem to be translated or mapped to a i18n string in SIW. Also do we want to call this  'Yubikey Security Method' ?

Reply:
This was a miss on my end, as I didn't map the label to the `select-authenticator-enroll` and `select-authenticator-authenticate` forms - thank you for catching that! Pushing up a new commit in a few to resolve it.

General rule I've been following is **if** the name of the authenticator is customizable/editable, then we wan't to leverage the API response in favor of relying on our own i18n translation. For this authenticator, it shouldn't be modifiable.

In regards to "Security Method" vs "Authenticator", we are only changing occurrences **where more than one authenticator** is being referenced. Here, this should remain `YubiKey Authenticator`.

Great questions!